son/interface.py

```diff
@@ -0,0 +1,92 @@
+"""son.interface
+
+Defines the public-facing interface for son. For performance reasons,
+son.read should be preferred over son.load.
+
+"""
+
+from pathlib import Path
+
+import son.stream as stream
+import son.serialize as serialize
+
+
+def dump(obj, file, is_metadata=False, **kwargs):
+    """dump an object to son file, metadata can only be written once
+
+    Args:
+        obj (Object): object to be dumped to string
+        file (str or Path): file to dump to
+        is_metadata (bool): is this a metadata entry?
+        **kwargs: args to be passed to son.dumps
```

Comment:
`son.dumps` doesn't exist anymore, right?

Reply:
it should be `son.serialize.dump`, yes. will fix